ws-security-dom/src/main/java/org/apache/wss4j/dom/message/WSSecSignature.java

```diff
@@ -898,6 +906,50 @@ private X509Certificate[] getSigningCerts() throws WSSecurityException {
         return certs;
     }
 
+    /**
+     * The method returns EdDSA signature algorithm for public key type (Ed25519 or Ed448).
+     *
+     * @param publicKey the public key to get the algorithm from
+     * @return the siganture algorithm (ED25519 or ED448) for the EdDSA public key
+     * @throws WSSecurityException if the algorithm cannot be determined
+     */
+    public String getSigAlgorithmForEdDSAKey(PublicKey publicKey) throws WSSecurityException {
```

Comment:
Could this be private static? I don't see a reason to make it public really.

Reply:
You are absolutely right. No need to be public. Now is fixed to private static.